media-plugins/distrho-ports/distrho-ports-9999.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# requires proaudio overlay
+
+EAPI=5
+
+inherit git-r3
+
+DESCRIPTION="Linux ports of Distrho plugins."
+HOMEPAGE="http://distrho.sourceforge.net"
+
+SRC_URI=""
+EGIT_BRANCH="master"
+EGIT_REPO_URI="https://github.com/DISTRHO/DISTRHO-Ports.git"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+
+IUSE=""
+
+DEPEND="virtual/jack
+	media-libs/freetype
+	<dev-util/premake-4.3"
```

Comment:
@audiodef These deps should all be defined. In case you're not sure it usually helps to search through the ebuilds in the gentoo tree
- https://github.com/gentoo/gentoo/search?l=Gentoo+Ebuild&q=freetype&type=&utf8=%E2%9C%93
- https://github.com/gentoo/gentoo/search?l=Gentoo+Ebuild&q=xrender&type=&utf8=%E2%9C%93
- https://github.com/gentoo/gentoo/search?l=Gentoo+Ebuild&q=xcursor&type=&utf8=%E2%9C%93

As @marcan said if they're optional they should be behind a USE flag to avoid automagic dependency usage.